packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -890,7 +942,11 @@ class MediaQuery extends InheritedWidget {
   static MediaQueryData of(BuildContext context) {
```

Comment:
Let's add some color to the documentation above explaining the difference between the general `of` method and the other more specific ones.